tflib/publisher/main.tf

```diff
@@ -43,6 +43,13 @@ data "oci_exec_test" "check-reproducibility" {
   timeout_seconds = 600
 }
 
+data "oci_exec_test" "check-killability" {
+  digest = module.this.image_ref
+  script = "docker kill $(docker run -d --rm $${IMAGE_NAME})"
```

Comment:
Are we sure that most containers will start a daemon process properly with the nested command?

Reply:
Oh, don't mistake my comment for criticism.  I love the idea of stuffing tests here, I am just worried a lot will: print usage and exit.

Maybe a middle ground is to create a common test module for this, and invoke it everywhere we can?